src/shared/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,39 @@
+import * as styles from './Popup.css';
+import PopIcon from '@/shared/assets/icon/ic_ wait_blue70_36.svg';
+import { useState } from 'react';
+
+const PopUp = () => {
+  const [isVisible, setIsVisible] = useState(true);
+  const handleClosePopup = () => {
+    setIsVisible(false);
+  };
+  return (
+    <div
+      className={styles.PopArea}
+      style={{ display: isVisible ? 'flex' : 'none' }}
```

Comment:
display none은 DOM에 계속 존재하고 있기 때문에 이벤트나 리소스를 차지할 수 있어요! 
조건부 렌더링으로 명확하게 처리해서 아예 DOM 자체에서 제거하는 것이 접근성이나 가독성 측면에서도 좋고 성능상 크게는 아니더라도 안전할 거 같다는 제안을.. 드려봅니당.  


```suggestion
  isVisible && (
      <div className={styles.PopArea}>
        <div className={styles.PopContents}>
          <img src={PopIcon} alt="팝업 아이콘" />
          <p className={styles.PopTitle}> 
          
          
          ... 
```